mgr/dashboard: Document custom RESTController endpoints (#25322)
ceph/ceph: src/pybind/mgr/dashboard/HACKING.rst

| DELETE       | Yes        | delete         | 204         |
+--------------+------------+----------------+-------------+

How to use a custom API endpoint in a RESTController?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

If you don't have any access restriction you can use ``@Endpoint``. If you

> have set a permission scope to restrict access to your endpoints,

Reviewer: ```suggestion
have set a permission scope to restrict the access to your endpoints,
```

Reviewer 2: Actually, "restrict access" is correct :)

Reviewer: Okay, thanks.

> ``@Endpoint`` fails, as it doesn't know which permission property should be

Reviewer: ```suggestion
``@Endpoint`` will fail, as it doesn't know which permission property should be
```

> used. To use a custom endpoint inside a restricted ``RESTController`` use

Reviewer: Just to remove the second 'use':

```suggestion
used. To use a custom endpoint inside a restricted ``RESTController`` choose
```
